### PR TITLE
PLAT-7961:Remove redundant code in BusinessProcessCaseService.php

### PR DESCRIPTION
**Pull request by david-winder-kaltura**

### CODE REVIEW

File: plugins/event_notification/providers/bpm/services/BusinessProcessCaseService.php [hunk 1]
```diff
@@ -116,9 +116,7 @@ public function serveDiagramAction($objectType, $objectId, $businessProcessStart
 		$filename .= $businessProcessStartNotificationTemplateId . '_';
 		$filename .= $caseId . '.jpg';
 		
-		$url = $provider->getCaseDiagram($caseId, $filename);
-		
-		KCurlWrapper::getDataFromFile($url, $filename);
```
Comment (MosheMaorKaltura) [inline plugins/event_notification/providers/bpm/services/BusinessProcessCaseService.php]: Why did you remove this line? it is needed.

Comment (david-winder-kaltura) [reply plugins/event_notification/providers/bpm/services/BusinessProcessCaseService.php]: Follow by discussion with Tan-Tan this can be remove . The second line getting an empty URL and therefor never set the file data

**Comment by MosheMaorKaltura** [comment]
approved
